05_wifi_virtualif.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash
+
+set -o nounset
+set -euo pipefail
+
+if [[ $EUID -ne 0 ]]; then
+	echo "This script must be run as root, use sudo $0 instead" 1>&2
+	exit 1
+fi
+
+## Install hostapd
+pacman -Syu --noconfirm hostapd
```

Comment:
Beware that here pacman is run with -u and --noconfirm options which means that all available system updates are installed without asking and python is updated to version 3.9!

Warning from Arch linux wiki: "Warning: When installing packages in Arch, avoid refreshing the package list without upgrading the system (for example, when a package is no longer found in the official repositories). In practice, do not run pacman -Sy package_name instead of pacman -Syu package_name, as this could lead to dependency issues."

Perhaps -Syu should be replaced with -S in this script.

Reply:
I tried running this script on a second host pc with manjaro last updated months ago (probably in September) and without -yu flags the hostapd package was not found at all.